src/tree/box.js

```diff
@@ -83,6 +84,12 @@ export default function box(
     .attr('style', d => `top:${y(d) - rootDiameter - buttonMargin}px;left:${x(d) + (cardWidth - rootDiameter) / 2}px`)
     .attr('id', d => d.data.id);
 
+  function getTooltipStyle(d) {
+    const halfCardWidth = cardWidth / 2;
+    const halfTooltipWidth = tooltipWidth / 2;
+    return `top:${(y(d)) * sel.transform.zoom + sel.transform.y - tooltipHeight - 25}px;left:${x(d) * sel.transform.zoom + sel.transform.x - (halfTooltipWidth - (halfCardWidth * sel.transform.zoom))}px;visibility: visible;opacity: 0.9;`;
```

Comment:
Since we are not using `tooltipHeight` anywhere else atm, set it to 60 + 25 and remove -25 here.